<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
<class>groupBoxStatistics</class>
<widget class="QGroupBox" name="groupBoxStatistics">
<property name="enabled">
<bool>false</bool>
</property>
<property name="geometry">
<rect>
    <x>80</x>
    <y>110</y>
    <width>226</width>
    <height>68</height>
</rect>
</property>
<property name="font">
<font>
    <pointsize>10</pointsize>
    <weight>75</weight>
    <bold>true</bold>
</font>
</property>
<property name="title">
<string>Statistics</string>
</property>
<layout class="QVBoxLayout" name="verticalLayout_10">
<item>
    <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
    <layout class="QVBoxLayout" name="verticalLayoutStatistics">
    <item>
        <layout class="QHBoxLayout" name="horizontalLayoutStatistics">
        <item>
        <widget class="QCheckBox" name="checkBoxStatistics">
        <property name="font">
            <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
            </font>
        </property>
        <property name="text">
            <string>histogram</string>
        </property>
        </widget>
        </item>
        <item>
        <widget class="QSpinBox" name="spinBoxStatistics">
        <property name="maximumSize">
            <size>
            <width>60</width>
            <height>16777215</height>
            </size>
        </property>
        <property name="font">
            <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
            </font>
        </property>
        <property name="suffix">
            <string>  bin</string>
        </property>
        <property name="prefix">
            <string/>
        </property>
        <property name="minimum">
            <number>1</number>
        </property>
        <property name="maximum">
            <number>1000000</number>
        </property>
        <property name="value">
            <number>10</number>
        </property>
        </widget>
        </item>
        <item>
        <spacer name="horizontalSpacer">
        <property name="orientation">
            <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
            <size>
            <width>40</width>
            <height>20</height>
            </size>
        </property>
        </spacer>
        </item>
        </layout>
    </item>
    <item>
        <widget class="QLabel" name="statisticsLabel">
        <property name="maximumSize">
        <size>
        <width>16777215</width>
        <height>0</height>
        </size>
        </property>
        <property name="font">
        <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        </font>
        </property>
        <property name="text">
        <string/>
        </property>
        </widget>
    </item>
    </layout>
    </item>
    </layout>
</item>
</layout>
</widget>
 <resources/>
 <connections/>
</ui>
